### PR TITLE
Unused variables

### PR DESCRIPTION
**Pull request by h3half**
Removes various unused variables across most of the project, as identified by SonarQube.

There are a few I left alone because their side-effects were important to the program.

### CODE REVIEW

**Comment by h3half** [comment]
I dunno why the 5 merges are there, they were from ``master`` in the first place. Help appreciated.

**Comment by gbasood** [comment]
Those merges don't hurt anything, so dont worry about the ones that are already there

File: SS14.Client.Graphics/Debug.cs [hunk 1]
```diff
@@ -60,7 +60,7 @@ public static void DebugRendertarget(RenderImage Rendertarget, string fileName)
         {
             string path = "..\\DEBUGTEXTURE\\" + fileName + ".png";
 
-           // Rendertarget.Texture.CopyToImage().SaveToFile(path);
+            Rendertarget.Texture.CopyToImage().SaveToFile(path);
```
Comment (gbasood) [inline SS14.Client.Graphics/Debug.cs]: ????

File: SS14.Client/Program.cs [hunk 2]
```diff
@@ -11,8 +11,7 @@ public class Program
         [STAThread]
         private static void Main()
         {
-            var args = Environment.GetCommandLineArgs();
-            GameController GC = new GameController();
+            GameController GC = new GameController(); // No, sonarqube, this isn't unused. A side-effect of this is to "run the game"
```
Comment (gbasood) [inline SS14.Client/Program.cs]: We can tell sonarqube to ignore individual issues in sonarqube itself

Comment (PJB3005) [reply SS14.Client/Program.cs]: Also, yes, it is unused, because the actual variable never gets used but the code will execute.

Comment (clusterfack) [inline SS14.Client/Program.cs]: The entire game runs from the constructor of the gamecontroller class?

Comment (ZoldorfTheWizard) [reply SS14.Client/Program.cs]: @clusterfack Yes. the game loop exists in the gamecontroller class (line 89).

Comment (PJB3005) [reply SS14.Client/Program.cs]: @h3half remove this and I'll merge it.

File: SS14.Shared/GameStates/GameState.cs [hunk 3]
```diff
@@ -146,7 +146,7 @@ public int WriteStateMessage(NetOutgoingMessage message)
         /// <returns></returns>
         public static GameState ReadStateMessage(NetIncomingMessage message)
         {
-            uint sequence = message.ReadUInt32();
+            uint sequence = message.ReadUInt32(); // This unused variable is vitally important, and the client will crash without it
```
Comment (PJB3005) [inline SS14.Shared/GameStates/GameState.cs]: Hah no it isn't.

You can discard return values you know.

**Pull request merged**